src/nvim/option_defs.h

```diff
@@ -329,7 +329,7 @@ EXTERN long p_ch;               /* 'cmdheight' */
 EXTERN int p_confirm;           /* 'confirm' */
 EXTERN int p_cp;                /* 'compatible' */
 EXTERN char_u   *p_cot;         /* 'completeopt' */
-EXTERN long p_ph;               /* 'pumheight' */
+EXTERN int p_ph;               /* 'pumheight' */
```

Comment:
You can't change this to `int`.
This is a global var for an option. The way options are implemented, all numeric options have to be the same type (currently `long`). We will eventually refactor that, because in fact, we want to get rid of type `long` (which is different size on Win64 than on other platforms), but it has to be done at the same time for all options. You can't just change one of them.